debian/control

```diff
@@ -29,7 +29,7 @@ Build-Depends: debhelper (>= 10),
                libboost-dev,
                libspdk-dev (>= 22.01.2),
                libipmctl-dev,
-               libraft-dev (= 0.10.1-1411.gefa15f4),
+               libraft-dev (= 0.11.0-1416.g12dbc15),
```

Comment:
A question only out of curiosity, possibly for @brianjmurrell 

I suppose that the "416" here refers to the 416th commit on https://daos-stack/daos/raft (i.e., commit [12dbc15](https://github.com/daos-stack/raft/commit/12dbc1595fad8b570de1e336205f994f2b0e22f5)

So in the future if daos were to update raft to a version 0.11.0-2 (and let's imagine that corresponds to commit 417 in daos-stack/raft, then what would this string be?  Would it be 0.11.0-2417.gxxxxxxx) or 0.11.0-1417.gxxxxxxx)?

Reply:
I don't know the true reason, but I've always assumed that this is the result of a typo.

Reply:
The 416 is the number of commits since a tag.  Ideally that number is not typically so high but we don't follow strict release practices on this repo so we are not tagging it on *release*s.  This is not terribly important though given that we don't really _release_ our raft but rather use it as a static library with DAOS.